README.md

```diff
@@ -133,6 +133,10 @@ Edit the `.params` file and add the required settings.
 
 #### `params.sample:`
```

Comment:
Here's where I think we're going to show `params.sample` and tell them to copy it and edit it.  If not, then there's a bit more to clean up.